torch/onnx/symbolic_opset9.py

```diff
@@ -307,7 +307,8 @@ def _export(name: str):
 
     def wrapper(func):
         globals()[name] = func
-        __all__.append(name)
+        global __all__
```

Comment:
Use `global` here in case of "Local variable defined in enclosing scope .." error